.github/workflows/deploy-infrastructure.yaml

```diff
@@ -38,7 +38,7 @@ on:
       workflow_timeout:
         description: "Timeout duration in minutes"
         required: false
-        default: 5
+        default: 10
```

Comment:
Genuine question.
I think changing the default means we wait upto 10 mins for every stack in every repo. But I think I was expecting to leave the default at 5 mins and then override it on  a stack-by-stack repo-by-repo basis as necessary. Eg if I am deploying cloudfront maybe I am prepared to wait 10 mins cos I know it can take time. But if I am deploying a stack that should take max two minutes but is hanging because of some unset variable do I really want to wait 10 mins.  What do we think?

Reply:
default increased to 10 because it's the only job in this pipeline currently